x-pack/plugins/security_solution/public/entity_analytics/pages/entity_analytics_management_page.tsx

```diff
@@ -13,15 +13,13 @@ import { RiskScoreEnableSection } from '../components/risk_score_enable_section'
 import { ENTITY_ANALYTICS_RISK_SCORE } from '../../app/translations';
 import { BETA } from '../../common/translations';
 import { RiskEnginePrivilegesCallOut } from '../components/risk_engine_privileges_callout';
-import { useIsExperimentalFeatureEnabled } from '../../common/hooks/use_experimental_features';
+import { useMissingRiskEnginePrivileges } from '../hooks/use_missing_risk_engine_privileges';
 
 export const EntityAnalyticsManagementPage = () => {
-  const privilegesCalloutEnabled = useIsExperimentalFeatureEnabled(
-    'riskEnginePrivilegesRouteEnabled'
-  );
+  const privileges = useMissingRiskEnginePrivileges();
```

Comment:
Elevate the privileges call to the page level as the sub components now need to use it